docs/how-to-guides/own_machine/clone-renku-project.rst

```diff
@@ -0,0 +1,116 @@
+.. _clone_renku_project:
+
+Clone a Renku project
+=====================
+
+In order to save your work back to RenkuLab from your own machine, you'll need to authenticate with RenkuLab.
+You can do this easily by the ``renku login`` command.
+
+However, if you find yourself needing to authenticate via an SSH Key, you can find those instructions below as well.
+
+.. tabbed:: Authenticate via renku login
+
+    To authenticate with Renku on a local machine where you have access to a browser, you can use :meth:`renku login <renku.ui.cli.login>`.
+
+    Provide the url of the RenkuLab instance where your project is located in the following command:
+
+    .. code-block:: console
+
+        $ renku login <url of your renkulab instance>
+
+    For example, if your project is on https://renkulab.io , enter:
+
+    .. code-block:: console
+
+        $ renku login renkulab.io
```

Comment:
```suggestion
        $ renku login --git renkulab.io
```

The `--git` flag is needed to modify the remote such that the token is used.

Reply:
erm, I apologize, I didn't fully realize the context of this line - of course `--git` doesn't work because in this how-to we are outside of a project by definition (setting up the credentials to clone it). So following these instructions will fail... indeed we need to remove the `--git` flag. Sorry!!